src/Uplink/Site/Data.php

```diff
@@ -155,7 +155,10 @@ public function get_domain( bool $original = false ): string {
 	protected function get_domain_multisite_option(): string {
 		/** @var string */
 		$site_url = get_site_option( 'siteurl', '' );
```

Comment:
```suggestion
		$site_url = get_site_option( 'siteurl', '' ) ?: get_site_url();
```

Could simplify it with that if you want, but either is fine for me.

Reply:
Makes sense great